pom.xml

```diff
@@ -29,12 +29,12 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.36</version>
+    <version>3.21</version>
     <relativePath />
   </parent>
 
   <artifactId>scm-api</artifactId>
-  <version>2.2.9-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

Comment:
Can we do incrementals in a separate PR

Reply:
I needed this to publish an incremental that I can consume in the downstream.

Reply:
Yep. Just put it in a separate PR and we’ll merge that first